catalog/spatial/registry/registry-federation-admin-service-impl/pom.xml

```diff
@@ -23,7 +23,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>registry-federation-admin-service-impl</artifactId>
-    <name>DDF :: Registry :: Federation Admin Service Impl</name>
+    <name>DDF :: Registry :: Federation :: Admin :: Service :: Impl</name>
```

Comment:
Not sure why you added the extra "::"s.  Those are usually map to the app and folder structure.